tests/test_AxiLiteRegs.py

```diff
@@ -0,0 +1,107 @@
+##############################################################################
+## This file is part of 'SLAC Firmware Standard Library'.
+## It is subject to the license terms in the LICENSE.txt file found in the
+## top-level directory of this distribution and at:
+##    https://confluence.slac.stanford.edu/display/ppareg/LICENSE.html.
+## No part of 'SLAC Firmware Standard Library', including this file,
+## may be copied, modified, propagated, or distributed except according to
+## the terms contained in the LICENSE.txt file.
+##############################################################################
+
+# dut_tb
+import cocotb
+from cocotb.clock    import Clock
+from cocotb.triggers import RisingEdge
+
+# test_AxiLiteRegs
+from cocotb_test.simulator import run
+import pytest
+import glob
+import os
+
+@cocotb.coroutine
+def dut_init(dut):
+
+    # Initialize the inputs
+    dut.axiClkRst.value     = 1
+
+    # Start clock (200 MHz) in a separate thread
+    cocotb.start_soon(Clock(dut.axiClk, 5.0, units='ns').start())
+
+    # Wait 1 clock cycle
+    yield RisingEdge(dut.axiClk)
+
+    # De-assert the reset
+    dut.axiClkRst.value = 0
+
+    # Wait 1 clock cycle
+    yield RisingEdge(dut.axiClk)
+
+@cocotb.coroutine
+def load_value(dut, ain, bin):
+
+    # Wait 1 clock cycle
+    yield RisingEdge(dut.axiClk)
+
+@cocotb.test()
+def dut_tb(dut):
+
+    # Initialize the DUT
+    yield dut_init(dut)
+
+    ###################################################
+    # Waiting for record type support for CocoTB + GHDL
+    ###################################################
+    # https://github.com/ghdl/ghdl/issues/2324
+    ###################################################
+
+    dut._log.info("DUT: Passed")
```

Comment:
This TB doesn't seem to actually verify anything.
I understand it's probably here as an example. Maybe the comments should be more explicit about this.

Reply:
This was a reminder that record types are not supported yet in CocoTB + GHDL. Do you want to remove this testbed and create a JIRA ticket as a reminder?

Reply:
[removing tests/test_AxiLiteRegs.py test](https://github.com/slaclab/surf/pull/1072/commits/04a1953c8fbbde7c71c0d153d23ea529b9bd66a2)